docs/_includes/sidebar.md

```diff
@@ -23,8 +23,8 @@
 <hr/>
 
 <ul class="sidebar-nav list-unstyled">
-  <li class="item"><a id=”side-nav-button-github” class="event-click" href="https://github.com/cyberark/conjur" target="_blank"><i class="fa fa-github-alt"></i> GitHub</a></li>
-  <li class="item"><a id=”side-nav-button-dockerhub” class=“event-click” href="https://hub.docker.com/r/cyberark/conjur/" target="_blank"><div class="icon-docker-hub"></div> DockerHub</a></li>
-  <li class="item coming-soon"><a id=”side-nav-button-cloud-formation” class=“event-click” href="#"><i class="fa fa-cloud"></i> Cloud Formation <span>(Coming Soon)</span></a></li>
-  <li class="item"><a id=”side-nav-button-slack” class=“event-click” href="https://slackin-conjur.herokuapp.com/" target="_blank"><i class="fa fa-slack"></i> Slack</a></li>
+  <li class="item"><a id="side-nav-button-github" class="event-click" href="https://github.com/cyberark/conjur" target="_blank"><i class="fa fa-github-alt"></i> GitHub</a></li>
+  <li class="item"><a id="side-nav-button-dockerhub" class="event-click" href="https://hub.docker.com/r/cyberark/conjur/" target="_blank"><div class="icon-docker-hub"></div> DockerHub</a></li>
+  <li class="item coming-soon"><a id="side-nav-button-cloud-formation" class="event-click" href="#"><i class="fa fa-cloud"></i> Cloud Formation <span>(Coming Soon)</span></a></li>
```

Comment:
Can we use this link text `AWS CloudFormation (coming soon)`?

that is how they say it: https://aws.amazon.com/cloudformation/

Reply:
yup! updating now